cmd/influxd/run/server.go

```diff
@@ -247,6 +252,7 @@ func (s *Server) appendRetentionPolicyService(c retention.Config) {
 	srv := retention.NewService(c)
 	srv.MetaClient = s.MetaClient
 	srv.TSDBStore = s.TSDBStore
+	srv.With(s.DiagnosticService.InRetentionContext())
```

Comment:
Any thoughts on removing the `In` prefix from these and just having 

```
srv.With(s.DiagnosticService.RetentionContext())
```

I think it reads pretty well.

Reply:
I like this.

Reply:
This meaning having the `In` prefix? I don't feel strongly one way or another 👍

Reply:
Oh no, I'm sorry. Removing the `In` prefix.